app/models/user.rb

```diff
@@ -105,7 +105,8 @@ def build_from_invitation(invitation)
     self.name = invitation.name
     self.email = invitation.email
     skip_confirmation!
-    course_users.build(course: invitation.course, name: invitation.name,
+    course_role = invitation.role || Course::CourseUser.roles[:student]
```

Comment:
I'd prefer if we kept this default convention in the `course_users` file, instead of having it in the `user`. In the future when we change defaults, it would be easier to locate these defaults. Maybe you can consider modifying the `set_defaults` method within the `course_user` model, or if you have a better method, you can do that!

Reply:
Changed in #2445